[Docs] 이슈 및 PR 템플릿 업데이트

DrunkenNeoguri opened the pull request:
## 📝 제목
<!-- PR에 대한 내용을 간략하게 제목으로 정의해주세요. -->
[Docs] 이슈 및 PR 템플릿 업데이트
  
## ✨ 작업 내용
<!-- 어떤 작업을 하셨는지 내용을 작성해주세요. -->
<!-- 작업된 화면이나 작업 전 후의 비교 이미지가 있으면 더더욱 좋아요. -->
- 프로젝트에서 사용할 이슈 및 PR 템플릿을 추가 및 수정하여 업데이트 했습니다.
  
## 💁‍♀️ 참고 사항
<!-- 작업하면서 참고하셨던 사항이 있거나, 읽으셨던 문서 등이 있으셨다면 여기에 남겨주세요. -->
<!-- 동료들에게 좋은 지식을 전파할 수 있는 기회에요 :D -->
- [리포지토리에 대한 문제 템플릿 구성](https://docs.github.com/ko/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
  
## 🤔 논의 사항
<!-- 작업하는 데에 있어서 고민되었던 내용이 있으셨나요? -->
<!-- 여기에 내용을 남겨주시면, 팀원들이 내용을 읽고 함께 논의해드릴 거예요. -->
- 깃헙 자동화를 테스트해볼 수 있는 좋은 기회네요 :D
  
## 🔧 앞으로의 과제
<!-- 작업 후, 남아있는 사항이나 기술 부채 등 작업한 내용과 연관되거나 해야할 사항이 있다면 남겨주세요. -->
<!-- 해당 내용들은 추후 새로운 태스크 등으로 등록하면서 관리하도록 해요. -->
- 메인으로 브랜치 올린 후에 직접적인 태스크 생성 및 프로젝트 연동이 되는지, 확인해봐야겠습니다.
  
## 🚨 관련 이슈
<!-- 작업과 관련하여 남아있는 이슈 등이 있다면 여기에 작성해주세요. -->
<!-- 작성하실 때는 [이슈 제목 #이슈번호](이슈 링크)로 남겨주세요. -->

DrunkenNeoguri committed:
🏗️Initial: Pull Request 템플릿 수정

DrunkenNeoguri committed:
🏗️Initial: Task, Bug, HotFix 이슈 템플릿 추가

hijjoy reviewed (approved):
확인했습니닷

i-nooo commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,29 @@
+<!-- 여기에 내용을 남겨주시면, 팀원들이 내용을 읽고 함께 논의해드릴 거예요. -->
+
+  
+## 🔧 앞으로의 과제
+<!-- 작업 후, 남아있는 사항이나 기술 부채 등 작업한 내용과 연관되거나 해야할 사항이 있다면 남겨주세요. -->
+<!-- 해당 내용들은 추후 새로운 태스크 등으로 등록하면서 관리하도록 해요. -->
```
개인적으로 앞으로의 과제는 불필요해보입니다!
논의사항으로 대체될 수 있을 것 같아보여서요.
이 부분은 추가 논의를 거쳐 결정하는 것도 좋을 것 같습니다.

hijjoy replied:
제 생각에도 chore, design, feat, refactor 등으로 세분화 하는 부분은 필요해 보입니다 !

DrunkenNeoguri replied:
[3321e30](https://github.com/Spoon-Lab/our-journey-fe/pull/2/commits/3321e30170f2b4a1962e7465e8dfe4009e1de00a) 버전에서 수정했습니다 ( _ _)

i-nooo commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,29 @@
+<!-- 해당 내용들은 추후 새로운 태스크 등으로 등록하면서 관리하도록 해요. -->
+
+  
+## 🚨 관련 이슈
+<!-- 작업과 관련하여 남아있는 이슈 등이 있다면 여기에 작성해주세요. -->
+<!-- 작성하실 때는 [이슈 제목 #이슈번호](이슈 링크)로 남겨주세요. -->
```
```
## 이슈 번호
- close #이슈번호
```
- 이런 식으로 관련 이슈 번호보다는 pr에 해당하는 이슈 번호만 적도록 유도하는 게 좋을 것 같습니다.
- 해당 부분을 최상단으로 올릴 수 있을까요?
- `#이슈번호`만 적어도 자동으로 해당 이슈 링크가 생성되기 때문에 별개의 하이퍼링크를 남기지 않아도 괜찮을 것 같아요!

DrunkenNeoguri replied:
[3321e30](https://github.com/Spoon-Lab/our-journey-fe/pull/2/commits/3321e30170f2b4a1962e7465e8dfe4009e1de00a) 버전에서 수정했습니다 ( _ _)

i-nooo reviewed (commented):
수고하셨습니다! 전체적으로 꼼꼼하게 작성하신게 보이네요 👍👍
다만, task로 뭉뚱그려 pr을 관리하기 보다는, chore, design, feat, refactor 등으로 세분화시키는 것이 좋을 것 같습니다.
개별적인 코멘트는 제 생각을 간단하게 작성한 것이니 반드시!! 해야만 한다는 아닙니다. 불필요하다면 언제든 스루해주셔도 좋아요:)

DrunkenNeoguri committed:
📚Docs: PR 리뷰 내용에 따른 Pull Request 템플릿 수정 #2

DrunkenNeoguri committed:
🗑️Remove: 이슈 방식 세분화에 따른 Task 템플릿 제거 #2

DrunkenNeoguri committed:
📚Docs: 논의 사항에 따른 각 케이스별 이슈 템플릿 추가 #2

DrunkenNeoguri committed:
📚Docs: Initial, Chore 템플릿에 체크리스트 및 예상 결과 항목 삭제 #2
- Initial은 초기 세팅에만 해당하는 부분이라 해당 항목이 필요 없을 듯 합니다.
- Chore 역시 패키지 설치나 설정만 변경하는 부분이라 불필요하다고 판단했습니다.

i-nooo reviewed (approved):
확인하였습니다! init, chore 삭제에 동의합니다 👍
템플릿 작성하신다고 고생하셨습니다 ㅠㅠ
